.github/workflows/config/spellcheck_config.yml

```diff
@@ -158,7 +158,7 @@ matrix:
       - open: '(?P<open>`+)'
         close: '(?P=open)'
       # Ignore words that start with an @, e.g. @param used in doc comments
-      - open: '@\S'
+      - open: '@\S+(?:\s[a-z]+)?'
```

Comment:
I believe this will remove spellchecking on the first word after `@param` tags, which is probably desirable, but it will also remove spellchecking for the first word after `@brief` tags, which is probably undesirable. Assuming the intent is to remove spellchecking on the parameter names, would it be better to make a new entry specifically for `@param`, like `'@param\s+(?:\S+)?'`, so that we don't miss misspelled words right after `@brief` tags?

Reply:
Sure. I would also prefer some uniformity regarding docs comments, like making them sentences that start with a captial letter (in which case the first word is also spell-checked), but this is not enforced by anything so far.